### PR TITLE
Ported FAT bucket to Open Liberty

### LINKED ISSUE #20193: Migrate com.ibm.ws.jaxws.tools_fat to Open Liberty
This is migration of com.ibm.ws.jaxws.tools_fat bucket

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #20193

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#build

**Review by neuwerk — APPROVED** [review]

**Comment by BerksanAtes** [comment]
#run-libby-bot

File: dev/io.openliberty.ws.jaxws.tools_fat/.settings/org.eclipse.jdt.ui.prefs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
...strumentation and @FFDCIgnore\n// http\://was.pok.ibm.com/xwiki/bin/view/Liberty/LoggingFFDC\n${exception_var}.printStackTrace();</template><template autoinsert\="true" context\="methodbody_context" deleted\="false" description\="Code in created method stubs" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.methodbody" name\="methodbody">// ${todo} Auto-generated method stub\n${body_statement}</template><template autoinsert\="true" context\="constructorbody_context" deleted\="false" description\="Code in created constructor stubs" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.constructorbody" name\="constructorbody">${body_statement}\n// ${todo} Auto-generated constructor stub</template><template autoinsert\="true" context\="getterbody_context" deleted\="false" description\="Code in created getters" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.getterbody" name\="getterbody">return ${field};</template><template autoinsert\="true" context\="setterbody_context" deleted\="false" description\="Code in created setters" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.setterbody" name\="setterbody">${field} \= ${param};</template></templates>
```
Comment (tevans78) [inline dev/io.openliberty.ws.jaxws.tools_fat/.settings/org.eclipse.jdt.ui.prefs]: You need to change the default java file template to have an OL copyright header and not an IBM Confidential one

**Review by tevans78 — DISMISSED** [review]
Need to change template copyright header

File: dev/io.openliberty.ws.jaxws.tools_fat/.settings/org.eclipse.jdt.ui.prefs (hunk 1 above)
Comment (tevans78) [reply dev/io.openliberty.ws.jaxws.tools_fat/.settings/org.eclipse.jdt.ui.prefs]: I suggest just copying this file from another OL project

Comment (BerksanAtes) [reply dev/io.openliberty.ws.jaxws.tools_fat/.settings/org.eclipse.jdt.ui.prefs]: Thank you!

Comment (BerksanAtes) [reply dev/io.openliberty.ws.jaxws.tools_fat/.settings/org.eclipse.jdt.ui.prefs]: That Copyright was very well hidden

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**